### DISCUSSION TITLE
Support the use of year, month and other functions in materialized views

### DISCUSSION THREAD

**Discussion by xiongzegang** [apache/doris#13084] [A - General / Q&A] [answered]
Hi doris team, 
Are there plans to support the use of year, month and other functions in materialized views。If there is such a plan, which version is it? Look forward to your reply.
请问doris开发组是否有计划在创建物化视图时支持year,month等日期函数？如果有这个计划，会在哪个版本支持？期待您的回复！

**Comment by BiteTheDDDDt** [comment, marked as answer]
这个计划是有的，不过目前还没有计划出明确的版本或者时间点，短期内（至少在1.2）应该是不会有这个功能。

**Reply by xiongzegang** [reply]
谢谢
